docs/book/src/user/quick-start.md

```diff
@@ -58,22 +58,6 @@ Test to ensure the local kind cluster is ready:
 kubectl cluster-info
 ```
 
-{{#/tab }}
-{{#tab v0.8.x}}
```

Comment:
Unfortunately, we cannot drop this paragraph entirely because older versions of CAPD (<= current) will continue to use network bridge 

So the tab should be preserved but with a different name (CAPD <= v0.3.X)

Reply:
Will update it back.